ui-spacetimechart/src/index.ts

```diff
@@ -3,3 +3,6 @@ import './styles/main.css';
 export * from './components/SpaceTimeChart';
 export * from './components/PathLayer';
 export * from './components/ConflictLayer';
+export * from './hooks/useCanvas';
```

Comment:
Why do you expose useCanvas ? I don't think that it is necessary

Reply:
yes, I removed